@codex plesae fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 7ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 5ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 5ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 6ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 4ms
 ✓ tests/prismSvelte.test.ts (1 test) 17ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 6ms
 ✓ tests/progressionBalance.test.ts (1 test) 33ms
 ✓ tests/domException.test.ts (1 test) 16ms
 ✓ tests/indexeddb.server.test.ts (1 test) 265ms
 ✓ frontend/tests/sanity.test.ts (1 test) 6ms
[warn] src/pages/quests/svelte/Quest.svelte
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 19082ms
   × Prettier formatting > frontend sources are formatted 19080ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 290 passed (291)
      Tests  1 failed | 1034 passed (1035)
   Start at  23:51:25
   Duration  168.20s (transform 10.12s, setup 11.33s, collect 46.55s, tests 111.44s, environment 190.58s, prepare 35.43s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9346, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/e2e/settings-page.spec.ts
+++ b/frontend/e2e/settings-page.spec.ts
@@ -42,17 +42,20 @@ test.describe('Settings route', () => {
             const uniqueCardRows = new Set(cards.map(toRoundedTop));
             const settingsRect = settingsContent.getBoundingClientRect();
             const legacyRect = legacyUpgrade.getBoundingClientRect();
-            const gridColumns = getComputedStyle(settingsContent)
-                .gridTemplateColumns.split(' ')
-                .filter(Boolean);
+            const settingsStyles = getComputedStyle(settingsContent);
+            const gridColumns = settingsStyles.gridTemplateColumns.split(' ').filter(Boolean);
+            const settingsContentLeft =
+                settingsRect.left + parseFloat(settingsStyles.paddingLeft || '0');
+            const settingsContentRight =
+                settingsRect.right - parseFloat(settingsStyles.paddingRight || '0');
 
             return {
                 rowCount: uniqueCardRows.size,
                 cardCount: cards.length,
                 gridColumnCount: gridColumns.length,
                 legacyGridColumn: getComputedStyle(legacyUpgrade).gridColumn,
-                settingsWidth: settingsRect.width,
-                legacyWidth: legacyRect.width,
+                legacyLeftGap: Math.abs(legacyRect.left - settingsContentLeft),
+                legacyRightGap: Math.abs(settingsContentRight - legacyRect.right),
             };
         });
 
@@ -60,9 +63,8 @@ test.describe('Settings route', () => {
         expect(desktopLayout?.gridColumnCount).toBeGreaterThan(1);
         expect(desktopLayout?.rowCount).toBeLessThan(desktopLayout?.cardCount ?? 0);
         expect(desktopLayout?.legacyGridColumn).toContain('1 / -1');
-        expect(desktopLayout?.legacyWidth ?? 0).toBeGreaterThanOrEqual(
-            (desktopLayout?.settingsWidth ?? 0) - 2
-        );
+        expect(desktopLayout?.legacyLeftGap ?? Number.POSITIVE_INFINITY).toBeLessThanOrEqual(4);
+        expect(desktopLayout?.legacyRightGap ?? Number.POSITIVE_INFINITY).toBeLessThanOrEqual(4);
 
         await page.setViewportSize({ width: 390, height: 844 });
         await page.reload();

